src/Product/Unit/Create.php

```diff
@@ -32,6 +32,10 @@ public function save(Unit $unit)
 
 	public function create(Unit $unit)
 	{
+		if (count($unit->options === 0)) {
```

Comment:
This breaks it. See the trello QA point. should be `count($unit->options) === 0`